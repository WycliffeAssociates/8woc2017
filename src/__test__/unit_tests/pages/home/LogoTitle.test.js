import React from "react";
import { shallow, render } from "enzyme";
import LogoTitle from "../../../../js/pages/home/LogoTitle";
import toJson from 'enzyme-to-json';

describe("Home Component, LogoTitle", () => {
	const wrapper = shallow(<LogoTitle />);
	const tree = toJson(wrapper);
	it("is rendered", () => {
		expect(wrapper).toHaveLength(1);
	});

	it('has no changes', () => {
		expect(tree).toMatchSnapshot();
	});
});
