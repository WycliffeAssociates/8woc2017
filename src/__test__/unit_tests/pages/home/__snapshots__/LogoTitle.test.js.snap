// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component, LogoTitle has no changes 1`] = `
<div>
  <Image
    as="img"
    fluid={true}
    src="trap.svg"
    ui={true}
  />
  <div
    className="pageTitle"
  >
    <font
      size="50"
    >
       
    </font>
  </div>
  <div
    className="logo"
  >
    <Image
      as="img"
      className="logoterra"
      src="logo.png"
      ui={true}
    />
  </div>
</div>
`;
